### PR TITLE
Remove old gloomtls and glooctl kube2e tests

### PR DESCRIPTION
**Pull request by npolshakova**
# Description
Remove old kube2e tests for gloomtls and glooctl. New gloomtls tests were introduced in https://github.com/solo-io/gloo/pull/9734. New glooctl tests were introduced in https://github.com/solo-io/gloo/pull/9718.

## API changes
None

## Code changes
None

## CI changes
- Remove old kube2e glooctl and gloomtls tests in favor of tests in new framework

## Docs changes
None

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: changelog/v1.18.0-beta10/remove-old-glooctl-gloomtls-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+changelog:
+  - type: NON_USER_FACING
+    issueLink: https://github.com/solo-io/solo-projects/issues/6303
+    resolvesIssue: false
+    description: >-
+      Remove old gloomtls and glooctl tests from kube2e to new framework.
+      skipCI-docs-build:true
```
Comment (sheidkamp) [inline changelog/v1.18.0-beta10/remove-old-glooctl-gloomtls-tests.yaml:7]: Nit: can we link the PRs where the new tests were introduced in the description?

**Review by sheidkamp — APPROVED** [review]
One requested update to the PR Description

**Review by sam-heilbron — APPROVED** [review]

**Review by jbohanon — APPROVED** [review]
:rocket:

**Comment by sam-heilbron** [comment]
bulldozer

**Pull request merged**